pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "spt3g"
+version = "0.1.0"
```

Comment:
This version number is not correct.  Should probably populate this using the same cmake utilities that are used to construct the version file that's stored in the python package.

Reply:
I can do that. Can you explain how the versioning works?

Reply:
One thing to note is that this string has to be https://peps.python.org/pep-0440/ compliant. So copying over existing stuff like
```
fullversion="autoprocessing-v3.3-203-ge4fce125"
```
unchanged won't work. `3.3.203` would work, but I'm not sure if that's everything you want, or how to produce a compliant string no matter the commit both on this public and on the private repo. Would definitely need help here, up to you how much effort you think is worth investing. In practice, unless people start really carefully making virtual environments with specific compatibility bounds on spt3g_software, I don't think the choice here affects anything else other than what you see from `pip list` or `poetry show`. E.g.:

Reply:
I believe the form 3.3.dev203 would be appropriate here.  I think it's pretty important to keep that 203 prominently available.

Reply:
Just pushed something that does this for Git. Actually on closer inspection you can also do a `+...` at the very end, where I put that hash, so its exactly that same info, e.g. `0.3.dev177+g77d5d4d9`. Could I ask for help for someone else to do the SVN branch? Or maybe we don't care about SVN?

Reply:
I think version numbers for the public repo are PEP-compliant numbers, so this should be fine.  The autoprocessing tags are only on the private 3G repo.